docs-client/src/containers/MethodPage/DebugPage.tsx

```diff
@@ -151,15 +160,19 @@ const DebugPage: React.FunctionComponent<Props> = ({
     let urlRequestBody;
     if (useRequestBody) {
       if (urlParams.has('request_body')) {
-        urlRequestBody = jsonPrettify(urlParams.get('request_body')!);
+        if (!isGraphqlService) {
```

Comment:
Q) not important, but wondering why prettify shouldn't be applied to `graphql` requests

Reply:
Checking again, this is unnecessary work. 🤣